### ISSUE TITLE
IFRAME elements are missing

### ISSUE THREAD

**Issue by fraseInc** [adbar/trafilatura#175]
I've noticed Iframe elements are missing, such as those containing youtube videos. I don't see any arguments to control iframes. Is this by design?

**Comment by adbar** [comment]
Hi @fraseInc, I tend indeed to discard iframes by design as embedded content is usually not as relevant text-wise. Do you have examples of elements which should be included?

**Comment by fraseInc** [comment]
I think things like Youtube, and videos in general are important. At least as an optional argument, similar to how you handle tables or images. we could add an include_videos filter?

The typical Youtube embed looks something like this:

`<iframe src="//www.youtube.com/embed/IkgLUo82eWg?rel=0" allowfullscreen="" loading="lazy" decoding="async"></iframe>`

**Comment by adbar** [comment]
An additional argument is tricky for maintenance reasons but thanks, I'll think about it.

**Comment by Manuel-reeporter** [comment]
have you added this functionality in the meantime?

**Comment by adbar** [comment]
The issue is still open and I'm available to review pull requests.

**Comment by adbar** [comment]
That's not correct, it's not an iframe in this example but (apparently) another issue.

**Comment by badrelmers** [comment]
Devtool shows it as iframe in my PC!
![Untitled](https://github.com/adbar/trafilatura/assets/40046489/2c6ad062-7240-4a0a-9d90-e1f6835d2307)

sometimes Medium allows me to copy paste all those iframe snippets by selecting all the article content (without having to copy it one by one), which means they were not iframes in such moments, so maybe Medium changes something from time to time (i did not test this theory i just noticed this behaviour).

**Comment by adbar** [comment]
Sorry, you're right, I thought it was a table but it's a whole HTML body contained in an iframe element... Tricky.
